src/Microsoft.Health.Dicom.Api.UnitTests/Extensions/MediaTypeHeaderValueExtensionsTests.cs

```diff
@@ -141,7 +142,7 @@ private static MediaTypeHeaderValue CreateMediaTypeHeaderValue(string mediaType,
 
         if (quality.HasValue)
         {
-            result.Parameters.Add(CreateNameValueHeaderValue(AcceptHeaderParameterNames.Quality, quality.ToString(), quoteValue: false));
+            result.Parameters.Add(CreateNameValueHeaderValue(AcceptHeaderParameterNames.Quality, quality.GetValueOrDefault().ToString(CultureInfo.InvariantCulture), quoteValue: false));
```

Comment:
Nullable<T> doesn't have a ToString method that accepts a culture, so it would have used the Current Culture by default. And I think semantically we want an Invariant culture